### ISSUE TITLE
Suggestion: Remove 'given, when, then, and' from method names 

### ISSUE THREAD

**Issue by jonphipps** [alnutile/pickle#9]
## Detailed description

One of the things that I'm finding is that when appending tests I have to be careful about the use of 'given, when, then, and' in Gherkin, because of the inclusion of those leading words in the test method name. I've taken to stripping those words from the generated methods and inserting them as comments:
```php
    public function testEditProfile() {
        $this->givenIHaveAProfileCreated();
        $this->andIamInEditMode();
        //etc etc
    }
```
becomes:
```php
    public function testEditProfile() {
        //given
        $this->IHaveAProfileCreated();
        //and
        $this->IamInEditMode();
        //etc etc
    }
```

## Context

Why is this change important to you? How would you use it?

It promotes reusability of methods across test scenarios and aligns more closely with the way Behat treats context methods.

**Comment by drbyte** [comment]
I like this suggestion. 
When doing TDD (and especially when debugging a failing test) I find I'm often rearranging my "then" and "and" lines, sometimes making them be "given" or "when" or "then". Behat allows this to be very fluid.

**Comment by defenestrator** [comment]
I agree with drbyte entirely. Cucumber and calabash have the same behavior, I believe. I love the idea of gherkin automated by dusk. I searched for it and found this repo. Super cool
